dodo.c

```diff
@@ -372,10 +375,31 @@ struct Instruction * parse_line(char *source, size_t *index){
         return 0;
     }
 
-    free(i);
+    /* ln where n is positive integer */
+    switch( source[*index] ){
+        case 'l':
+        case 'L':
+            ++(*index);
+            break;
+        default:
+            printf("parse_line: unexpected character '%c', expected 'l'\n", source[*index]);
+            free(i);
+            return 0;
+            break;
+    }
+
+    ret = parse_number(i, source, index);
+
+    if( i->argument.num == 0 ){
```

Comment:
thinking again about the idea of `line 0` I think we should allow it, even though it will be the same as `byte 0`.

so we should be able to just remove this test.